src/languages/java.js

```diff
@@ -5,7 +5,7 @@ Category: common, enterprise
 Website: https://www.java.com/
 */
 
-export default function(hljs) {
+export default function (hljs) {
```

Comment:
Did the linter flag this?  pretty sure I turned that off as we prefer no space.

Reply:
My VSCode did this, when I hit the save button I think...

Reply:
If you can configure it to not rewrite the whole file (for the future) we'd appreciate it.